src/annotations/components/annotationForm/AnnotationStartTimeInput.tsx

```diff
@@ -30,6 +46,24 @@ export const AnnotationStartTimeInput: FC<Props> = (props: Props) => {
     }
   }
 
+  const isInputValueInvalid = (): boolean => {
```

Comment:
when designing methods, it's typically better name them and design them in the affirmative, not the negative. that way, if you need to negate the value, you don't have a double negative in the middle of your code.

so instead of

```ts
isInputValueInvalid()
!isInputValueInvalid()
```
you'd have
```ts
isInputValueValid()
!isInputValueValid()
```

secondly: this is relying on `inputValue` existing and being in scope. it's generally better method design to pass in dependencies and return some computation. if we ever need to move this method or reuse it, it becomes much more portable:

```ts
isInputValueValid(inputValue: string): boolean => {

// then later when it's called

if (!isInputValueValid(inputValue)) {
  return ...
}